scripts/sanityCheck.js

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env babel-node
+
+import { Log, txUtils, env, eth, contracts, cli } from 'decentraland-commons'
+import { db } from '../src/database'
+import { Parcel, ParcelService } from '../src/Parcel'
+import { Publication } from '../src/Publication'
+import { BlockchainEvent } from '../src/BlockchainEvent'
+import { asyncBatch } from '../src/lib/asyncBatch'
+import { loadEnv } from './utils'
+import { decodeAssetId } from './monitor/utils'
+
+const log = new Log('update')
+
+let BATCH_SIZE
+
+export async function sanityCheck() {
+  log.info(`Using ${BATCH_SIZE} as batch size, configurable via BATCH_SIZE`)
+
+  const shouldFix = await cli.confirm('Fix errors?')
+
+  log.info('Connecting database')
+  await db.connect()
+
+  log.info('Connecting to Ethereum node')
+  await eth.connect({
+    contracts: [contracts.LANDRegistry, contracts.Marketplace],
+    providerUrl: env.get('RPC_URL')
+  })
+
+  log.info('[Check 1/2]: Checking Marketplace sanity against all events')
+  try {
+    await marketplaceCheck(shouldFix)
+  } catch (err) {
+    console.log(err, err.stack)
+  }
+
+  log.info('[Check 2/2]: Checking parcel ownership from -150,-150 to 150,150')
+  await processParcels(shouldFix)
+}
+
+async function marketplaceCheck(fix) {
+  const marketplace = eth.getContract('Marketplace')
+  const events = await getAllEvents(marketplace)
+  const filterParcels = getParcelIds(events)
```

Comment:
I find this a bit confusing. `getParcelIds` doesn't return ids but an object, and when I tried to see how it's used in the loop below, the name of the iteration variable is `d` which didn't help me either.

If you agree, maybe something like:

```javascript
const filterParcels = getEventParcels(events)

const tasks = []
for (let { parcelId, lastEvent } of filterParcels) {
  tasks.push(checkParcel(parcelId, lastEvent, marketplace, fix))
}
```

is a bit more descriptive? it's hard to name!